src/python_testing/TC_TSTAT_4_2.py

```diff
@@ -356,8 +356,9 @@ async def test_TC_TSTAT_4_2(self):
                 logger.info(
                     "Couldn't run test step 4 since there were no built-in presets")
 
-            # Send the SetActivePresetRequest command
-            await self.send_set_active_preset_handle_request_command(value=b'\x03')
+            notBuiltInPresets = list(preset for preset in current_presets if preset.builtIn is False)
+            if len(notBuiltInPresets) > 0:
+                activePreset = notBuiltInPresets[0]
```

Comment:
I don't get either the old code (which does not match testplan) nor the new code.  Specifically:

1. After this point we do not use activePreset, except in cases where we set it to something else.
2. We also do not use `notBuiltInPresets` without first resetting it.

So what is this chunk doing. What am I missing?

The next bit that sets activePresetHandle also does not make sense....

Reply:
The deleted lines should have been removed in the previous PR, and then when I merged from master there was a conflict that I somehow resolved with some lines from the next test. Removed.